src/main/java/org/codehaus/plexus/archiver/Archiver.java

```diff
@@ -458,4 +462,14 @@ ResourceIterator getResources()
      * @since 4.2.0
      */
     String getOverrideGroupName();
+
+    /**
+     * Configure the archiver to get reproducible archives (see <a href="https://reproducible-builds.org/>Reproducible
+     * Builds</a>). This will require reproducible archive entries order, defined entries timestamp and reproducible
+     * entries Unix mode.
+     * 
+     * @param outputTimestamp the date to use for archive entries last modified time
+     * @since 4.2.0
+     */
+    void configureReproducible( Date outputTimestamp );
```

Comment:
Why did you call it `outputTimestamp` if you desribe it with "last modified time"? Why not not used `lastModifiedDate`/`lastModifiedTimestamp` right away?

Reply:
good idea: renaming to lastModifiedDate